cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilderTest.java

```diff
@@ -1410,6 +1411,122 @@ public void testOAuthOptions()  {
         assertThat(actualOptions, is(equalTo(expectedOptions)));
     }
 
+    @ParallelTest
+    public void testCustomConfigSetProtocolMapCorrectlyForsSslSasl() {
+        GenericKafkaListener listener = new GenericKafkaListenerBuilder()
+                .withName("CUSTOM-LISTENER")
+                .withPort(9092)
+                .withType(KafkaListenerType.INTERNAL)
+                .withTls(true)
+                .withNewKafkaListenerAuthenticationCustomAuth()
+                .withSasl(true)
+                .withListenerConfig(ImmutableMap.of())
```

Comment:
Please use `Map.of()` instead (here and in other places) or build the Map separately (it does not need to be immutable). This seems to pull Guava dependency which we don't really want just because of `ImmutableMap`. This is also what failed the CI build.